.gitignore

```diff
@@ -1,2 +1,13 @@
 # emacs
-*~
\ No newline at end of file
+*~
+
+# pycharm
+.idea/
+
+.ipynb_checkpoints/
+limnos.egg-info/
+limnos/.ipynb_checkpoints/
+limnos/Untitled.ipynb
```

Comment:
Why is `Untitled.ipynb` explicitly ignored? Nothing in limnos generates such a file (right?)